datumaro/components/format_detection.py

```diff
@@ -32,6 +32,7 @@ class FormatDetectionConfidence(IntEnum):
     belonging to the detector's format.
     """
 
+    EXTREAME_LOW = 5
```

Comment:
Could you fix a typo? `EXTREME_LOW`

Reply:
Why this should be required?

Reply:
This is a scrap during development. I'll remove it.

Reply:
Removed.